test/ApplicationTest.php

```diff
@@ -68,6 +68,26 @@ public function testRouteMethodReturnsRouteInstance()
         $this->assertSame($this->noopMiddleware, $route->getMiddleware());
     }
 
+    public function testAnyRouteMethod()
+    {
+        $this->router->addRoute(Argument::type(Route::class))->shouldBeCalled();
+        $route = $this->getApp()->any('/foo', $this->noopMiddleware);
+        $this->assertInstanceOf(Route::class, $route);
+        $this->assertEquals('/foo', $route->getPath());
+        $this->assertSame($this->noopMiddleware, $route->getMiddleware());
+
+        $reflClass    = new \ReflectionClass($this->getApp());
```

Comment:
@spiffyjr I agree with @samsonasik - but have a different suggestion:

- Import `ReflectionProperty` into the file.
- Use the following:
  ```php
  $app = $this->getApp();
  $r = new ReflectionProperty($app, 'httpRouteMethods');
  $r->setAccessible(true);
  $routeMethods = $r->getValue($app);
  ```

Since you're interested in a specific property, do reflection directly on it, instead of pulling it from class reflection. :smile: